May Serenity update - 2-way associations (#604)
akeneo/pim-helpcenter: content/updates/2020-05/2-2-way-associations.md

# Product associations update

> ## New association type: the 2-way association

Reviewer: I'm not comfortable with having only one subsection under the main title.
If there is only one new feature regarding the associations in May, I'd suggest removing putting only one main title.

Reviewer: @LaureBro, what do you think?

Author: OK, I'll make a new PR.

> ::: meta-data type="New" features="Product associations, Productivity, Data Governance" available="mid-May" link-to-doc="../articles/manage-your-association-types.html"

Reviewer: ```suggestion
::: meta-data type="New" features="Product associations, Productivity, Data Governance" available="mid-May" link-to-doc="../articles/manage-your-association-types.html#create-a-2-way-association-type"
```
I would better put this link.

:::

> We created a new association type named **2-way association**, accessible from `Settings` then `Association types` tabs.

Reviewer 2: ```suggestion
We created a new association type named **2-way association**. You can find it in `Settings` then `Association types`.
```

> When associating some products to others, you now have the possibility to link them via a two-way association.

Reviewer 2: ```suggestion
When associating some products with others, you now have the possibility to link them thanks to a two-way association.
```

> Example: if you associate product A to product B, product B will be **automatically** associated to product A.

Reviewer 2: ```suggestion
Example: if you associate product A to product B, product B will **automatically** be associated with product A.
```

> This feature is especially useful for clients who need to display which products are compatible with which other products (like machines and spare parts).

Reviewer 2: ```suggestion
This feature is particularly useful for clients who need to display products that are compatible with other products (like machines and spare parts).
```

![2-way association type](../img/Associations_2-way-association-type.png)

::: more

> [What are association types?](../articles/manage-your-association-types.html) or 

Reviewer: ```suggestion
[What are association types?](../articles/manage-your-association-types.html)  
```

Author: I added "or" because otherwise it displays the articles next to each others, and we barely see they are 2 different articles. There's no line break :/

Reviewer: There is a line break if you put two spaces at the end of the first line. :wink: 
It's Markdown classical behavior.

Author: Oh, thanks Nolwenn.
